content/providers/05-observability/langfuse.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Langfuse
+description: Monitor, evaluate and debug LLM applications with Langfuse
+---
+
+# Langfuse Observability
+
+[Langfuse](https://langfuse.com/) ([GitHub](https://github.com/langfuse/langfuse)) is an open source LLM engineering platform, that helps teams to collaboratively develop, monitor and debug AI applications. Langfuse integrates with the Vercel AI SDK to provide:
+
+- Application [traces](https://langfuse.com/docs/tracing)
+- Usage patterns
+- Cost data by user and model
+- Replay sessions to debug issues
+- [Evaluations](https://langfuse.com/docs/scores/overview)
+
+## Setup
+
+<Note>
+  You need to be on `"ai": "^3.3.0"` to use the telemetry feature as it was
```

Comment:
something like:
"To use telemetry feature, ensure you are using version 3.3 of the AI SDK or later"